commercial/app/model/commercial/events/LiveEvent.scala

```diff
@@ -0,0 +1,51 @@
+package model.commercial.events
+
+import model.commercial.events.Eventbrite._
+import org.joda.time.DateTime
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Reads}
+
+case class LiveEventImage(id: String, mainImageUrl: String)
```

Comment:
Perhaps call this ID `eventId` to make it clear that it's the ID of the event not of the image?